packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/LightCurveViewer/LightCurveViewer.js

```diff
@@ -18,9 +19,19 @@ const ZOOM_OUT_VALUE = 0.8
 const ZOOMING_TIME = 100  // milliseconds
 
 function storeMapper (stores) {
-  const { setOnZoom } = stores.classifierStore.subjectViewer
+  const {
+    annotate,  // bool: Indicates if the Classifier is in Annotate mode
+    move,  // bool: Indicates if the Classifier is in Annotate mode
+    setOnZoom,  // func: sets onZoom event handler
+  } = stores.classifierStore.subjectViewer
+  
+  let interactionMode = ''
+  if (annotate && !move) interactionMode = 'annotate'
+  if (!annotate && move) interactionMode = 'move'
```

Comment:
Instead of determining the interaction mode here, you could define a views function in the store that returns a computed value (see: https://github.com/mobxjs/mobx-state-tree#views as well an example of a couple in the `ClassificationStore`), so that way you can just map the computed value from the store to the component directly and have the logic be handled by the store. 

Note: I consider this a non-blocking comment. This can be addressed in a follow up PR.

Reply:
If it makes more sense, I'm happy to change the logic to use `props.annotate` instead of `interactionMode`

i.e. change it from `if (interactionMode === '...') {}` to simply `if (annotate) { /*blah*/ } else { /*assume 'move' by default*/ }`

The reason I spelled out the interaction mode here is because the `SubjectViewer` stores the `annotate` and `move` booleans separately for some reason. While it SHOULD operate on either-or logic, I wasn't certain if I need to account for when both `move` and `annotate` are true, or both are false.

Reply:
OK, I can make that work in this PR - moving the interactionMode 'calculus' to SubjectViewerStore is easy. My initial concern was that 'annotate' and 'move' could still operate independently of one another, but...

> return 'annotate' // maybe default to annotate mode?

...putting a default value in the calculus makes all the sense I need. 👌